moped-editor/src/views/projects/projectView/ActivityLogComponents/ProjectActivityEntry.js

```diff
@@ -0,0 +1,149 @@
+    },
+    project_website: {
+      label: "project website",
+    },
+    knack_project_id: {
+      label: "knack internal ID",
+    },
+    // deprecated column, but keeping because historical activities depend on it
+    purchase_order_number: {
+      label: "purchase order number",
+    },
+    task_order: {
+      label: "task order",
+    },
+    // deprecated column, but keeping because historical activities depend on it
+    work_assignment_id: {
+      label: "work assignment ID",
+    },
+    parent_project_id: {
+      label: "parent project id",
+    },
+  },
+};
+
+const useStyles = makeStyles((theme) => ({
+  entryText: {
+    padding: "0 0 0 .5rem",
+  },
+  boldText: {
+    fontWeight: 700,
+  },
+  strikeText: {
+    textDecoration: "line-through",
+  },
+}));
+
+const getEntryText = (change, legacyVersion, classes) => {
+  // adding a field
+  if (change.description[0].old === null) {
+    return (
+      <Typography variant="body2" className={classes.entryText}>
+        Added {entryMap.fields[change.description[0].field].label}{" "}
+        {change.description[0].new}
+      </Typography>
+    );
+  }
+
+  // updating a field
+  if (legacyVersion) {
+    return (
+      <Typography variant="body2" className={classes.entryText}>
+        Changed {entryMap.fields[change.description[0].field].label} from "
+        <span className={classes.strikeText}>{change.description[0].old}</span>"{" "}
+        to "{change.description[0].new}"
+      </Typography>
+    );
+  } else {
+    return (
+      <Typography variant="body2" className={classes.entryText}>
+        Changed {entryMap.fields[change.description[0].field].label} from "
+        <span className={classes.strikeText}>
+          {change.description[0].old[change.description[0].field]}
+        </span>
+        " to "{change.description[0].new[change.description[0].field]}";
+      </Typography>
+    );
+  }
+};
+
+const ProjectActivityEntry = ({ change }) => {
+  const classes = useStyles();
+
+  let changeData = {};
+  // the legacy way has the change data in event/data.
+  // In the new schema, event is undefined
+  const legacyVersion = !!change.record_data.event;
+
+  if (legacyVersion) {
+    changeData = change.record_data.event.data;
+  } else {
+    changeData = change.record_data.data;
+  }
+
+  // project creation
+  if (change.description.length === 0) {
+    return (
+      <Box display="flex" p={0}>
+        <Box p={0}>
+          <BeenhereOutlinedIcon />
+        </Box>
+        <Box p={0} flexGrow={1}>
+          <Typography variant="body2" className={classes.entryText}>
+            {changeData.new.project_name} created
+          </Typography>
+        </Box>
+      </Box>
+    );
+  }
+
+  return (
+    <Box display="flex" p={0}>
+      <Box p={0}>
+        <span class="material-symbols-outlined">summarize</span>
+      </Box>
+      <Box p={0} flexGrow={1}>
+        {getEntryText(change, legacyVersion, classes)}
+      </Box>
+    </Box>
+  );
```

Comment:
Is the idea that you can't get here? Of if we do, we've managed to pull up a project which is no longer active?

I guess I'm not sure what the meaning of `summarize` is, but I'm not sure what I need to do to a project to reach this branch.